### ISSUE TITLE
the problem in the "main_topcoder.py "

### ISSUE THREAD

**Issue by yunyouke** [lezasantaizi/Spoken-language-identification#2]
in the line 127 : "gru_output = F.dropout(gru_output, training=self.training)" , the  " F.dropout" is better to be changed to "nn.Dropout". The dropout module nn.Dropout conveniently handles this and shuts dropout off as soon as your model enters evaluation mode, while the functional dropout does not care about the evaluation / prediction mode. It can be referred to https://stackoverflow.com/questions/53419474/using-dropout-in-pytorch-nn-dropout-vs-f-dropout
